docs/user-manuals/fine-grained-cpu-orchestration.md

```diff
@@ -0,0 +1,138 @@
+        - name: NodeNUMAResource
+          args:
+            apiVersion: kubescheduler.config.k8s.io/v1beta2
+            kind: NodeNUMAResource
+            # the CPU binding policy ('None', 'FullPCPUs', 'SpreadByPCPUs', 'ConstrainedBurst')
+            preferredCPUBindPolicy: FullPCPUs
+            # the NUMA allocation strategy ('MostAllocated', 'LeastAllocated', 'DistributeEvenly')
+            numaAllocateStrategy: MostAllocated
+            # the scoring strategy
+            scoringStrategy:
+              # the kind of scoring strategy ('MostAllocated', 'BalancedAllocation', 'LeastAllocated')
+              type: MostAllocated
+              # the weights of each resource type
+              resources:
+              - name: cpu
+                weight: 2
+              - name: memory
+                weight: 1
+        plugins:
+          # enable the NodeNUMAResource plugin
+          preFilter:
+            enabled:
+              - name: NodeNUMAResource
+          filter:
+            enabled:
+              - name: NodeNUMAResource
+              ...
+          score:
+            enabled:
+              - name: NodeNUMAResource
+                weight: 1
+              ...
+          reserve:
+            enabled:
+              - name: NodeNUMAResource
+          preBind:
+            enabled:
+              - name: NodeNUMAResource
+```
+
+The koord-scheduler takes this ConfigMap as [scheduler Configuration](https://kubernetes.io/docs/reference/scheduling/config/).
+New configurations will take effect after the koord-scheduler restarts.
+
+## Use Fine-grained CPU Orchestration
+
+1. Create an `nginx` deployment with the YAML file below. Set QoS classes of pods as [`LSE`](/docs/architecture/qos#definition) or [`LSR`](/docs/architecture/qos#definition).
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-lsr
+  labels:
+    app: nginx-lsr
+    koordinator.sh/qosClass: LSR
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx-lsr
+  template:
+    metadata:
+      name: nginx-lsr
+      labels:
+        app: nginx-lsr
+        koordinator.sh/qosClass: LSR # set the QoS class as LSR
+    spec:
+      schedulerName: koord-scheduler # use the koord-scheduler
+      containers:
+      - name: nginx
+        image: nginx
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 500m
+      priorityClassName: koord-prod
+```
+
+2. Deploy the `nginx` deployment and check the scheduling result.
+
+```bash
+$ kubectl create -f nginx-deployment.yaml
+deployment/nginx-lsr created
+$ kubectl get pods -o wide | grep nginx
+nginx-lsr-59cf487d4b-jwwjv   1/1     Running   0       21s     172.20.101.35    node-0   <none>         <none>
+nginx-lsr-59cf487d4b-4l7r4   1/1     Running   0       21s     172.20.101.79    node-1   <none>         <none>
+nginx-lsr-59cf487d4b-nrb7f   1/1     Running   0       21s     172.20.106.119   node-2   <none>         <none>
+```
```

Comment:
Add the strategy description of physical core spreading or binpack?

Reply:
added